projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/yes-no/write-yes-no-field.component.ts

```diff
@@ -19,4 +18,8 @@ export class WriteYesNoFieldComponent extends AbstractFieldWriteComponent implem
   public ngOnInit() {
     this.yesNoControl = this.registerControl(new FormControl(this.yesNoService.format(this.caseField.value))) as FormControl;
   }
+
+  public toLowerCase(value: string) {
```

Comment:
@MateiHMCTS - is this function used anywhere? Couldn't spot it but I may have missed it. Also, it's missing a `void` return type